src/js/Badges/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Props } from '../index';
+
+interface BadgeProps extends Props {
+  badgeStyle?: React.CSSProperties;
+  badgeClassName?: string;
+  badgeId: string | number;
+  children: React.ReactNode;
```

Comment:
Whoops... I thought I fixed all of them after you mentioned that about the `TableColumn`. Thanks! The link and example definitely clears up why it is not ideal.